src/Layout/QueryParamContext.tsx

```diff
@@ -562,6 +573,60 @@ const QueryParamProvider: FC = ({ children }) => {
     setContactDetails(cleanDetails);
   };
 
+  const getDonationBreakdown = useCallback(() => {
+    if (!paymentSetup || !isSupportedDonation || !tenant) {
+      const totalAmount = paymentSetup ? paymentSetup.unitCost * quantity : 0;
+      return {
+        mainProjectAmount: totalAmount,
+        supportAmount: 0,
+        totalAmount: totalAmount,
```

Comment:
Since the property name and variable name are the same, you can write `totalAmount `directly here using the object property shorthand.